source/index.md

```diff
@@ -946,6 +946,29 @@ Coming soon...
 We like to experiment in the Node.js ecosystem, and we're promoting single page applications as the way to buld rich experience for the web. This tutorial doesn't cover that aspect. If you want to learn more, please check our [tutorial](https://blog.cozycloud.cc/post/2015/09/29/Build-and-share-your-single-page-app-with-React%2C-Node-and-Pouchdb%2C-part-I)!
 
 
+## Package a static application for installation into your cozy platform
+
+<br style="clear: both;" />
+
+  ```json
+  {    
+    "name": "cozy-[name of your app]", 
+    "description": "[Description of your app : it will appear on your cozy-home platform]",
+    "cozy-type": "static", 
+    "cozy-displayName": "[The name you want displayed on your cozy-home platform]", 
+    "icon-path": "[the path of the icon you want to use for your static app ('path/to/icon.png')"
+  }
+  ```
+  
+To deploy a static application into your cozy platform, you need to do the following steps :
```

Comment:
* can you reuse the formula "package a static app for installation into your Cozy platform" here?
* typo: "steps:" (no space before ":")